be/src/exec/hdfs_scanner_orc.cpp

```diff
@@ -488,7 +488,38 @@ Status HdfsOrcScanner::do_open(RuntimeState* runtime_state) {
             auto* split_context = down_cast<const SplitContext*>(_scanner_ctx.split_context);
             options.setSerializedFileTail(*(split_context->footer.get()));
         }
+        DataCacheHandle cache_handle;
+        std::string metacache_key;
+        if (_cache) {
```

Comment:
look at 3 lines above? 
maybe we have set serialized file tail.

Reply:
done